adobe/react-spectrum
# RSP-1862 [Docs][A11y] SideNav and ToC should have aria-current for current page and location

In Documentation site,

1. SideNavItem link for current page should have `aria-current="page"`.
2. ToC link for current location within the page should have `aria-current="location"`.


Closes [RSP-1862](https://jira.corp.adobe.com/browse/RSP-1862)
## ✅ Pull Request Checklist:

- [x] Included link to corresponding [RSP-1862](https://jira.corp.adobe.com/browse/RSP-1862).

## 📝 Test Instructions:

1. Run `yarn start:docs`, open http://localhost:1234/react-aria/useListBox.html
2. Verify that Side Nav link for the current page has `aria-current="page"`.
3. Verify that the ToC link for the current location has `aria-current="location"`.
4. Scroll the page so ToC updates current location and verify that the ToC link for the current location has updated to include `aria-current="location"`.
5. Navigate to a new page, verify that Side Nav link for the new page has `aria-current="page"`.
6. Navigate to an index page, verify that no Side Nav link has `aria-current="page"`.

## 🧢 Your Project:

Accessibility
